### ISSUE TITLE
Training loss curve

### ISSUE THREAD

**Issue by sungnyun** [facebookresearch/DiT#17]
Hello authors. 
I appreciate you for this nice work!

I'm currently training the DiT model, given the code you have uploaded a few days ago.
I followed the exactly same training script, except for that I changed the model from DiT-XL/2 to DiT-S/8.

The problem is that the training loss does not decrease after a few hundres of steps. 
(from around 500 steps, avg_loss converged to ~0.2)
Is this a normal behavior, or is it because of the small model architecture?
Have you ever encountered with this issue? If so, how can I avoid this?

Thanks in advance.

**Comment by wpeebles** [comment]
Hi @sungnyun, S/8 should decrease beyond 0.2 loss. In general diffusion loss curves can be pretty noisey (and smaller DiT models will have train losses that decrease more slowly than the larger DiTs), but I'd expect S/8 to have a train loss consistently around ~0.19 by like 5000 updates (and eventually it should decrease to ~0.17 although it may take up to like 100K updates to get there). Would you mind sharing your training log?

**Comment by wpeebles** [comment]
I trained an S/8 model from scratch last night on two GPUs for ~75K updates. Command:

The training curve appears correct to me up to this point. If you plot the "Train Loss" values logged to console, here's the curve you should expect:

![image](https://user-images.githubusercontent.com/35885960/218238342-a5108992-c6d1-4266-bd0c-bdb5ae200646.png)

I'll go ahead and close this issue now, but if you encounter any problems feel free to reopen or make a new issue!

**Comment by sungnyun** [comment]
Hi @wpeebles, thank you for the help:) 
I watched the loss up until 10k steps, but I guess I should've trained way longer. 
Now that I've trained until 30k steps, I'm seeing the loss fluctuating btw 0.187~0.19, so it seems similar to yours.
